crates/neon/src/lifecycle.rs

```diff
@@ -54,6 +58,36 @@ pub(crate) struct InstanceData {
 
     /// Shared `Channel` that is cloned to be returned by the `cx.channel()` method
     shared_channel: Channel,
+
+    /// Table of user-defined global cells.
+    globals: GlobalTable,
+}
+
+pub(crate) struct GlobalTable {
+    cells: Vec<Option<Box<dyn Any + Send>>>,
```

Comment:
Does this _need_ to be `Send`? I'm not actually clear if it does since theoretically it is only available on the same thread. If we don't want to think about it too hard, we could relax this bound in the future.

I do think it needs to be `'static`.

Reply:
We discussed `Send` on a call and it's hard to predict. You could imagine a future Node runtime that runs workers on green threads and might execute the same worker on different OS threads at different times. But it's hard to know if even that is unsafe since it'd still be sequential (maybe if it depends on TLS?) and it might not even be a likely possibility. All the same, it just feels like we should do the safer thing for now and we could consider relaxing it in the future.